### PR TITLE
Changes the amount of conveyor belts needed for a Clarke from 4 to 5

### PR DESCRIPTION
**Pull request by SeigaSeiga**
## About The Pull Request

Increases the amount of conveyor belts needed for Clarke construction from 4 to 5, an negligible iron cost increase of about 1.5 sheets.

## Why It's Good For The Game

I understand the idea of it being an even number of belts. It implies symmetry. But it's just annoying to work with. Nearly every  other mech construction step in the game either asks for one of something, or a multiple of five. Protolathes and autolathes are best suited to print single items, or multiples of 5, 10, etc. This change would indirectly streamline the process of building the mech.

I'm so tired of hitting the x5 button to get the required conveyor belts in a stack and then having this single leftover belt with a MASSIVE sprite sitting around in the mech lab. I mean yeah I could send it down disposals and make it someone elses problem. But I'd rather just not have to deal with it in the first place.

## Changelog
:cl:
qol: Changed the number of Conveyor Belts required for a Clarke from 4 to 5 to match how most machines produce item stacks.
/:cl:

### CODE REVIEW

**Comment by improvedname** [comment]
could always make the clarke parts a lil cheaper counting the cost increasement for the fifth conveyor belt

**Comment by Jacquerel** [comment]
I don’t think this is making it **functionally** more expensive, the interface is slow enough that i can’t imagine anyone prints one belt four times instead of five belts once.

**Comment by Sealed101** [comment]
pretty negligible concern, but this is some very wacky reasoning considering that both the autolathe and as of late the protolathe have a custom print amount option
also this means you have to print at least one belt on icebox to make a clarke

**Comment by SyncIt21** [comment]
Have you checked my PR #71774 before considering this?

**Comment by SeigaSeiga** [comment]
> Have you checked my PR #71774 before considering this?

I had no idea that was a thing. I watched the video; that plays like a hidden feature rather than an obvious function of the UI. But I am forced to accept that printing a custom amount is indeed a thing.

I stick by the consistency and streamlining reasoning.

**Comment by SyncIt21** [comment]
> > Have you checked my PR #71774 before considering this?
> 
> I had no idea that was a thing. I watched the video; that plays like a hidden feature rather than an obvious function of the UI. But I am forced to accept that printing a custom amount is indeed a thing.
> 
> I stick by the consistency and streamlining reasoning.

The title of the video should have given you the hint before you even played the video.
My PR voids your streamline reasoning as it makes production faster unlike this PR which. ages production costlier for no reason

**Comment by SeigaSeiga** [comment]
> The title of that PR should have given you the hint before you even played the video.

Of course it did. The video was actually the thing that made me realise how easy it is to miss the feature during the natural course of playing the game. Even assuming that it _was_ obvious to players, it's still a less streamlined input than hitting the more obvious x5 and x10 buttons. And because of that...

>> Protolathes and autolathes are best suited to print single items, or multiples of 5, 10, etc
> Should also be deleted because that's false now as it can print any amounts between 1 & 50

Protolathes and autolathes are STILL best suited to printing multiples of 5 or 10. They can do any amount, but they're best suited for it, because it's still the more obvious functionality to anyone not fully initiated. Pretending that this isn't the case is silly.

I still fully believe that this would be good for consistency and benefit roboticists in the long run by making the construction less tedious for players who aren't aware of these protolathe features, but now that _I_ know about the feature I guess the perceived problem has been solved. Knowledge gaps just kind of exist as part of SS13's design, so I'll just take what I've learned and leave new players to figure it out for themselves as well.

>also this means you have to print at least one belt on icebox to make a clarke
You're telling me that dismantling your workplace is standard for mech construction? That's ... weird to me. But to each their own.

----

TL;DR I wrote the original PR with a lot more dramatic flair than it really deserved and I actually don't think this is a big deal. Good job on the new protolathe feature #71774. Your feature is good even if the way you argue is by confidently asserting you've found a contradiction where there isn't one. But point taken, this change ain't necessary. Into the trash it goes!
